src/Umbraco.Cms.Integrations.DAM.Aprimo/readme.md

```diff
@@ -78,5 +78,12 @@ Properties available from the strongly-typed model:
 - Crops
 - Asset fields
 
+### Working with Crops
```

Comment:
Some code samples would be useful here I think (and also for the "Working with fields") section below.

Please also can you make sure any updates here are added to the pending PR on the docs platform?  Just whilst we're migrating over we need to update in both places.

Reply:
Updated.